src/components/Popover/index.tsx

```diff
@@ -47,10 +62,12 @@ function Popover(props) {
                 onClose={onCloseWithPopoverContext}
                 type={CONST.MODAL.MODAL_TYPE.POPOVER}
                 popoverAnchorPosition={anchorPosition}
-                animationInTiming={disableAnimation ? 1 : animationInTiming}
-                animationOutTiming={disableAnimation ? 1 : animationOutTiming}
+                animationInTiming={disableAnimation ? 1 : animationInTiming ?? 1}
+                animationOutTiming={disableAnimation ? 1 : animationOutTiming ?? 1}
```

Comment:
Why do you need nullish coalescing operator here? If these params are optional maybe you can add default values to them?